index.html

```diff
@@ -404,6 +404,9 @@ <h2 id="Intro">1 Introduction</h2>
           <a href="#AC">W3C Advisory Committee</a>), the Team, and the general public.</li>
       </ol>
 
+      <p>The W3C also operates Community and Business Groups,
+      which are separately described in <a href="https://www.w3.org/community/about/agreements/">their own process document</a>.
```

Comment:
Should that be incorporated by reference, making it a normative part of the process?

Reply:
No, I believe it should not be. Additional incorporations would trigger even more up-front legal review, making more of a barrier to entry for even those not intending to do CG/BG work.